public/scss/pages/ShippingMethod.module.scss

```
@@ -58,14 +58,20 @@
       {
         content: "";
         @include fixedSize(12px);
-        @include position(absolute, 50%, auto, auto, 15%);
+        @include position(absolute, 50%, auto, auto, 17%);
         transform: translateY(-50%);
         display: block;
         border-radius: 100%;
         background: $color_black;
       }
     }
   }
+
+  &[class*=disabled]
```

```suggestion
  &[class*="disabled"]
```

alangkah baiknya dikasih kutip dua mba